docs-src/modules/ROOT/pages/connect/resolvers.adoc

```diff
@@ -256,6 +256,30 @@ of resolvers called by providing some base information that you already have, yo
   [:customer/full-name]}]
 ----
 
+== Alias resolvers
+
+Sometimes you want to make an alias, that is, create a resolver that just converts one
```

Comment:
Sometimes you might want to make an alias, that is, create a resolver which just converts one
name to another. For example: